daceml/onnx/op_implementations/fpga_implementations.py

```diff
@@ -2154,6 +2157,15 @@ def forward(node: ONNXOp, state: SDFGState,
         X = in_desc_with_name(node, state, sdfg, "X")
         Y = out_desc_with_name(node, state, sdfg, "Y")
         vec_width = X.veclen
+        out_vec_width = Y.veclen
+
+        # Xilinx implementation specific
+        vendor = dace.config.Config.get("compiler", "fpga_vendor")
+        xilinx = True if vendor == "xilinx" else False
```

Comment:
We discussed before that querying the vendor at configuration time is not really kosher. If you already discussed this with @TizianoDeMatteis then it's okay?

Reply:
It's queried at expansion time. Either this or you need two different expansions (one for Intel, one for Xilinx). To me, it's ok to do it like Manuel did

Reply:
Yes exactly, I think the only alternative would be two different expansions. I had the feeling this is cleaner than making to separate versions, which share so much code except for the single buffer of different types.

Reply:
I think the correct thing to do is having two expansions :-) But whatever, this is fine for this purpose